### DISCUSSION TITLE
Attempted import error: 'Text' is not exported from '@nextui-org/react' (imported as 'Text').

### DISCUSSION THREAD

**Discussion by SadikAli-Karadiya** [heroui-inc/heroui#2614] [Feedback]
Why i am getting this error?

**Comment by theonlyway** [comment]
Where's your code? NextUI v2 (Which is the latest version) doesn't even have a `Text` component.

**Comment by SadikAli-Karadiya** [comment]
import { Tooltip, Button, Grid, Avatar } from ***@***.***/react";
import { UserTwitterCard } from "./UserTwitterCard";
import { DeleteUser } from "./DeleteUser";

export default function App() {
  return (
    <Grid.Container gap={2} alignItems="center">
      <Grid>
        <Tooltip trigger="click" content={<DeleteUser />}>
          <Link>
            <Text b color="error">
              Click Delete User
            </Text>
          </Link>
        </Tooltip>
      </Grid>
      <Grid>
        <Tooltip placement="top" content={<UserTwitterCard />}>
          <Avatar
            pointer
            size="lg"
            src="https://i.pravatar.cc/150?u=a042581f4e29026704d"
            color="gradient"
            bordered
            squared
          />
        </Tooltip>
      </Grid>
    </Grid.Container>
  );
}

I have copied this code from below link

https://v1.nextui.org/docs/components/tooltip

**Comment by theonlyway** [comment]
Did you install v2 or v1. That’s the v1 documentation but the latest version is v2 and it doesn’t have a Text component

**Comment by SadikAli-Karadiya** [comment]
No, I have only installed @nextui-org/react.

So, which package should I have to install to access Text, Grid and Row
components?

**Reply by jrgarciadev** [reply]
Those components were removed when we migrated to v2, please check this:

Text (needless) - @see https://tailwindcss.com/docs/font-size
Container (needless) - @see  https://tailwindcss.com/docs/container
Grid (needless) - @see  https://tailwindcss.com/docs/display#grid

**Reply by ketangupta34** [reply]
Could you please provide a good reason why these components were removed? If I am importing a UI library, I would want the text component to handle the text in my app. I don’t want to have to create custom CSS just for Text. This would be a deal breaker for me in using Next UI now. Moreover, the NavBar in Next UI is now containerized, but there is no container component provided. This means I have to manually ensure the paddings match the NavBar, which is another deal breaker.

**Comment by SadikAli-Karadiya** [comment]
Does this means that the documentation on NextUI website is based on older
version?

**Reply by theonlyway** [reply]
No. I don’t know how you landed on the v1 documentation. I go to the NextUi website and the default documentation is for v2 https://nextui.org/docs/components/tooltip

**Reply by ketangupta34** [reply]
by default from google OR any search we land on V1 documentation only. something Next UI should fix

**Comment by SadikAli-Karadiya** [comment]
Is there any way in v2 to open tooltip on click instead of hover?

**Reply by theonlyway** [reply]
Have you tried using an onClick event with useState to set the value of isOpen?

I would hazard you want something more like this though 

```
<Popover>
     <PopoverTrigger>
          <Info size={16} />
      </PopoverTrigger>
       <PopoverContent>
       This is the difference between this history entry and
                        the previous one
       </PopoverContent>
 </Popover>
```
